pkg/rulefmt/rulefmt.go

```diff
@@ -33,48 +33,70 @@ type Error struct {
 	Group    string
 	Rule     int
 	RuleName string
-	Err      error
+	Err      WrappedError
+}
+
+// WrappedError wraps error with the yaml node which can be used to represent
+// the line and column numbers of the error.
+type WrappedError struct {
+	err     error
+	node    *yaml.Node
+	nodeAlt *yaml.Node
 }
 
 func (err *Error) Error() string {
-	return errors.Wrapf(err.Err, "group %q, rule %d, %q", err.Group, err.Rule, err.RuleName).Error()
+	if err.Err.nodeAlt != nil {
+		return errors.Wrapf(err.Err.err, "%d:%d: %d:%d: group %q, rule %d, %q", err.Err.node.Line, err.Err.node.Column, err.Err.nodeAlt.Line, err.Err.nodeAlt.Column, err.Group, err.Rule, err.RuleName).Error()
+	} else if err.Err.node != nil {
+		return errors.Wrapf(err.Err.err, "%d:%d: group %q, rule %d, %q", err.Err.node.Line, err.Err.node.Column, err.Group, err.Rule, err.RuleName).Error()
+	}
+	return errors.Wrapf(err.Err.err, "group %q, rule %d, %q", err.Group, err.Rule, err.RuleName).Error()
 }
 
 // RuleGroups is a set of rule groups that are typically exposed in a file.
 type RuleGroups struct {
 	Groups []RuleGroup `yaml:"groups"`
 }
 
+// RuleGroupsTest for running tests over rules.
+type RuleGroupsTest struct {
```

Comment:
This is unused.

Reply:
This is actually used inside the `manager_test.go`. It helps to convey `yaml.Node` type into default strings type so that Marshalling can be done according to the tests.

Reply:
If it's only used in tests, then it belongs in the test file.

Reply:
Sure. I will update that.

Reply:
This still seems to be here